security/pkg/nodeagent/sds/sdsservice_test.go

```diff
@@ -33,6 +33,8 @@ import (
 	"google.golang.org/genproto/googleapis/rpc/status"
 	"google.golang.org/protobuf/testing/protocmp"
 
+	ca2 "istio.io/istio/pkg/security"
```

Comment:
why not simply use the package name `security` here? and the meaning of `ca2` is not quite clear to me.

Reply:
Sorry, didn't notice it - I generally let IDE fill everything. Will fix.